js_modules/dagster-ui/packages/ui-core/src/integrations/__generated__/dbt.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * THIS FILE IS GENERATED BY `yarn generate-integration-docs`.
+ *
+ * DO NOT EDIT MANUALLY.
+ */
+
+import {IntegrationFrontmatter} from '../types';
+import dbtLogo from './logos/dbt.svg';
+
+export const logo = dbtLogo;
+
+export const frontmatter: IntegrationFrontmatter = {
+  id: 'dbt',
```

Comment:
We have 2 entries for dbt. I think ultimately we would want dbt and dbt cloud.

Reply:
Oh yeah, there's nothing even in this one. Is there anything I can reliably look at in the source md that will tell me that I can skip it?

Reply:
This appears to be a case where we don't want to include the `index.md` because it only contains a `DocCardList` component. But there's no front-matter to indicate that, since some `index.md` are valid.